js/web/lib/wasm/jsep/webgpu/ops/common.ts

```diff
@@ -215,8 +215,14 @@ export interface IndicesHelper {
   readonly shape: readonly number[];
 }
 
-const getWgslValueType = (type: number, components: 1|2|3|4): string|[string, string] => {
-  // return type is [ storage type, runtime type ] or a single string for both
+/**
+ * A helper function to get [ storage type, runtime type ] or a single string for both.
+ *
+ * @param type - the tensor type.
+ * @param components - indicates the number of components of each element. 1 for scalar, 2 for vec2, 3 for vec3, 4 for
+ *    vec4.
+ */
+export const getWgslValueType = (type: number, components: 1|2|3|4): string|[string, string] => {
```

Comment:
Please set a default value to components, such as `export const getWgslValueType = (type: number, components: 1|2|3|4 = 1)`

Reply:
I think it is a code style issue. It seems that onnxruntime webgpu backend does not always use default parameter, and I do not know whether the api owner likes the changes. Let us keep it here and hear maintainers’ views.

Reply:
please merge to latest and this function is already added into `common.ts`.